python/GafferSceneUI/LightEditor.py

```diff
@@ -359,6 +359,15 @@ def __editSelectedCells( self, pathListing, quickBoolean = True ) :
 						inspections.append( ( column.inspector(), inspection, parentInspection ) )
 
 		if len( inspections ) == 0 :
+			# \todo Add a way to identify columns that inspect attributes so this warning
```

Comment:
I think we can already identify attribute-inspecting columns with `isinstance( column, InspectorColumn ) and isinstance( column.inspector(), AttributeInspector )`. But I don't understand what we're trying to do here - what am I missing?

Reply:
There's a situation where you won't be able to toggle because we couldn't find any inspectors. My intention is to give a popup message to that effect to but now I'm trying to recall where that situation comes up. I'll think more on it tonight.

Reply:
Right, the case I am catching is, for example, trying to toggle a mute on a group location with the Edit Scope set to `None`. Without this, it will do nothing and not give any feedback as to why. 

Since my original code comment is incorrect, and we can find out if the column is an attribute column, we could :
1. Give the popup warning for any attribute column
2. Give the same popup for any column, including shader parameters, even though those can never be set for non-lights
3. Give the popup for attributes, and a different worded one for parameters
4. Output a warning to the terminal instead
5. Remove this entirely

I favor 1, but am open to other opinions.

Reply:
Do you prefer 1 because you don't see the need for 2 because nobody could reasonably expect that to _ever_ work, and they therefore don't need a warning? That sounds reasonable, but I'd like to play devil's advocate. We don't know what inspectors we might introduce in future, and eventually we intend to allow custom inspectors to be defined as extensions. And it might not be _completely_ obvious to a user why they shouldn't be able add a parameter on a group. So maybe the simpler version 2 where we warn for everything has some merit?

Reply:
> because you don't see the need for 2 because nobody could reasonably expect that to _ever_ work, and they therefore don't need a warning

Yeah, that's where I was coming from, it felt a bit noisy to me. But considering the plans for more columns and customization, the second option seems like a nice helping-hand. I can also imagine it handy for someone clicking / entering a row above or below what they really wanted, and a popup would help then too.